srcpkgs/python3-debugpy/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'python3-debugpy'
+pkgname=python3-debugpy
+version=1.5.1
+revision=1
+archs="x86_64* i686"
```

Comment:
This is likely due to cross (not arch) and not really an option, we want to be able to keep building ipykernel for arm.

Something in the build is going wrong and making it use host tools rather than cross ones.